build.gradle

```
@@ -5,7 +5,7 @@ plugins {
 }
 
 group 'uk.gov.hmcts.reform'
-version '1.4.3'
+version '1.4.4'
```

This should really be a MINOR bump because we're adding a new feature/event. PATCH increases should be reserved for fixing bugs.
```suggestion
version '1.5.0'
```